x-pack/test/functional/apps/lens/heatmap.ts

```diff
@@ -74,8 +73,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.lens.openPalettePanel('lnsHeatmap');
       await testSubjects.setValue('lnsPalettePanel_dynamicColoring_stop_value_0', '10', {
         clearWithKeyboard: true,
+        typeCharByChar: true,
       });
-      await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
I wonder if we should think at some point to replace this in all places - it was giving flakiness in many places so far. Just a comment, nothing about this PR :)

Reply:
Well yes it makes sense actually :)